internal/cli/alerts/acknowledge.go

```diff
@@ -44,14 +44,18 @@ func (opts *AcknowledgeOpts) initStore() error {
 	return err
 }
 
+var ackTemplate = `ID	TYPE_NAME	METRIC_NAME	ACK_UNTIL
+{{.ID}}	{{.EventTypeName}}	{{.MetricName}}	{{.AcknowledgedUntil}}
```

Comment:
this is an "action" kind of operation, similar to what we were doing with delete, and now with create and update we can make this be something like
```suggestion
var ackTemplate = "Alert {{.ID}} acknowledged until {{.AcknowledgedUntil}}"
```

or something like that